### PR TITLE
Add COC

### PR DESCRIPTION
**Pull request by rayandas**

### CODE REVIEW

File: src/code-of-conduct.md [hunk 1]
```diff
@@ -3,68 +3,83 @@ layout: content.liquid
...
 
-### Contact Information
+Feel free to contact a conference volunteer to find the Code of Conduct committee or directly reach out to the Code of Conduct committee and they will take your report and do the needful action. Volunteers will be wearing "PyCon India Volunteer" badges and CoC committee will be wearing a special colour shirt to be able to easily identify them. Please be assured that any report/incident you shared with us will be highly confidential and will not leave outside the bounds of CoC committee members.
 
-If you are being harassed, notice that someone else is being harassed, or have any other concerns, please contact a member of conference staff. Conference staff will be wearing "PyCon India Volunteer" badges.
+Contact any of the following committee members. The Helpdesk can assist you in finding them at the venue
 
-Or send an email to [coc@in.pycon.org](mailto:coc@in.pycon.org)
+For more information on how to report incidents and what to include in your report, please visit our [Reporting Guide](https://in.pycon.org/2024/). It provides clear instructions on the reporting process and the steps we take to address reported incidents.
```
Comment (anistark) [inline src/code-of-conduct.md]: Update the link?

Comment (rayandas) [reply src/code-of-conduct.md]: Updated.

File: src/code-of-conduct.md [hunk 2]
```diff
@@ -3,68 +3,83 @@ layout: content.liquid
...
+* Attempt to resolve differences of opinion peacefully.
+* Avoid harassment, exclusionary or offensive behaviour of any kind. This includes but is not limited to offensive verbal comments related to gender, sexual orientation, disability, physical appearance, body size, race, religion, sexual images in public spaces, deliberate intimidation, stalking, following, harassing photography or recording, sustained disruption of talks or other events, inappropriate physical contact, and unwelcome sexual attention.
 
-**Code for Speakers:**
+Violations can be reported online by sending a mail to [coc@in.pycon.org](mailto:coc@in.pycon.org), or to PyCon India volunteers at the venue.
 
-Sexual language or imagery is inappropriate for your talks or slides. Refrain from using sexist, racist or exclusionary language anywhere in your content.
+If a participant engages in behaviour that violates this code of conduct, the Code of Conduct team may take any action they deem appropriate, including warning the offender or expulsion from the community and community events with no refund of event tickets. Please read on for a more detailed version of the Code of Conduct below.
```
Comment (sayanchowdhury) [inline src/code-of-conduct.md]:
```suggestion
If a participant engages in behaviour that violates this Code of Conduct, the Code of Conduct team may take any action they deem appropriate, including warning the offender or expulsion from the community and community events with no refund of event tickets. Please read on for a more detailed version of the Code of Conduct below.

```

File: src/code-of-conduct.md [hunk 3]
```diff
@@ -3,68 +3,83 @@ layout: content.liquid
...
+* Offensive verbal comments related to gender, sexual orientation, disability, physical appearance, body size, race, religion.
+* Violent threats or language directed against another person.
+* Swearing, insults or discriminatory/exclusionary/offensive jokes or language.
+* Deliberate intimidation, stalking, following, harassing photography or recording.
+* Sustained disruption of talks or other events.
+* Using sexually explicit or violent material in presentations. Sexual language or imagery is inappropriate for all aspects of the conference, including talks and exhibits.
+* Unwelcome sexual attention or inappropriate physical contact.
+* Advocating for, or encouraging, any of the above behaviour.
+* Repeated harassment of others. In general, if someone asks you to stop, then stop.
+* Avoid use of sexualized clothing/uniforms/costumes, or otherwise create a sexualized environment.
```
Comment (sayanchowdhury) [inline src/code-of-conduct.md]: Add a point that when interacting with other people, to not demean anyone and maintain a polite tone towards others.

**Comment by abhishekmishragithub** [comment]
LGTM! :shipit:

**Review by sayanchowdhury — APPROVED** [review]

**Pull request merged**